AUTHORS

```diff
@@ -701,7 +701,6 @@ Mathias Küsel <mathiask@hotmail.de>
 Qasim Zaidi <qasim@zaidi.me>
 Sam Newman <newmansam@outlook.com>
 Zach Bruggeman <mail@bruggie.com>
-Michaël Zasso <mic.besace@gmail.com>
```

Comment:
Any reason for moving duplicates to the bottom? The list should be ordered, afaik, and this commit actually moves some contributors down.